test/arrays/deitz/many/test_many_arrays_of_records.chpl

```diff
@@ -12,7 +12,9 @@ record R2 {
 
 for param i in 1..p {
   var A1: [1..2] R1(i);
-  writeln(A1);
+  writeln("(p = ", A1[1].p, ", x = ", A1[1].x,
+          ") (p = ", A1[2].p, ", x = ", A1[2].x, ")");
```

Comment:
Yes, but for simplicity, I just modified all of the writeln's in the same way for this kind of test... It was an easier change. I don't think it matters, for the intent of the test.